lua/dashboard/theme/hyper.lua

```diff
@@ -187,8 +187,10 @@ local function mru_list(config)
     local filename = vim.fn.fnamemodify(file, ':t')
     local icon, group = utils.get_icon(filename)
     icon = icon or ' '
-    if not utils.is_win then
-      file = file:gsub(vim.env.HOME, '~')
+    if config.mru.cwd_only then
```

Comment:
why we need colon here. just doc ci failed can ignore.

Reply:
You mean next line? 
`file = vim.fn.fnamemodify(file, ':.')` ?? 

> :.	Reduce file name to be relative to current directory, if
		possible.  File name is unmodified if it is not below the
		current directory.

https://neovim.io/doc/user/cmdline.html#%3A%3A.

Reply:
yes, everything seems to be working as expected.